examples/demo-atmos.d/custom-import/extra-config.yaml

```diff
@@ -0,0 +1,2 @@
+logs:
```

Comment:
Rename this file from `extra-config.yaml` to `logging.yaml` and move all log-levels here.